packages/twenty-server/src/engine/api/graphql/graphql-config/graphql-config.module.ts

```diff
@@ -1,11 +1,10 @@
 import { Module } from '@nestjs/common';
 
 import { CoreEngineModule } from 'src/engine/core-modules/core-engine.module';
-import { graphQLFactories } from 'src/engine/api/graphql/graphql-config/factories';
 
 @Module({
   imports: [CoreEngineModule],
-  providers: [...graphQLFactories],
-  exports: [...graphQLFactories],
+  providers: [],
+  exports: [],
 })
 export class GraphQLConfigModule {}
```

Comment:
nit: Is this module still needed now? Should we import CoreEngineModule directly in the module importing GraphQLConfigModule?

Reply:
Merging the PR to move forward as it's been approved but let's look at that in the upcoming "Twenty Apps" PR